code/modules/events/spacevine.dm

```diff
@@ -145,17 +147,53 @@
+
+/// Hurts mobs. To be used when a vine with aggressive spread mutation spreads into the mob's tile or buckles them.
+/datum/spacevine_mutation/aggressive_spread/aggrospread_act(obj/structure/spacevine/S, mob/living/M)
+	var/mob/living/carbon/C = M //If the mob is carbon then it now also exists as a "C", and not just an M.
+	if(istype(C)) //If the mob (M) is a carbon subtype (C) we move on to pick a more complex damage proc, with damage zones, wounds and armor mitigation.
+		var/obj/item/bodypart/limb = pick(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG, BODY_ZONE_HEAD, BODY_ZONE_CHEST) //Picks a random bodypart. Does not runtime even if it's missing.
+		var/armor = C.run_armor_check(limb, "melee", null, null) //armor = the armor value of that randomly chosen bodypart. Nulls to not print a message, because it would still print on pierce.
+		var/datum/spacevine_mutation/thorns/T = locate() in S.mutations //Searches for the thorns mutation in the "mutations"-list inside obj/structure/spacevine, and defines T if it finds it.
+		if(T && (prob(40))) //If we found the thorns mutation there is now a chance to get stung instead of lashed or smashed.
+			C.apply_damage(50, BRUTE, def_zone = limb, wound_bonus = rand(-20,10), sharpness = SHARP_POINTY) //This one gets a bit lower damage because it ignores armor.
+			C.Stun(1 SECONDS) //Stopped in place for a moment.
+			playsound(get_turf(M), 'sound/weapons/pierce.ogg', 50, TRUE, -1)
+			M.visible_message("<span class='danger'>[M] is nailed by a sharp thorn!</span>", \
+			"<span class='userdanger'>You are nailed by a sharp thorn!</span>")
+			log_combat(S, M, "aggressively pierced") //"Aggressively" for easy ctrl+F'ing in the attack logs.
+		else
+			if(prob(80))
+				C.apply_damage(60, BRUTE, def_zone = limb, blocked = armor, wound_bonus = rand(-20,10), sharpness = SHARP_EDGED)
+				C.Knockdown(2 SECONDS)
+				playsound(get_turf(M), 'sound/weapons/whip.ogg', 50, TRUE, -1)
+				M.visible_message("<span class='danger'>[M] is lacerated by an outburst of vines!</span>", \
+				"<span class='userdanger'>You are lacerated by an outburst of vines!</span>")
+				log_combat(S, M, "aggressively lacerated")
+			else
+				C.apply_damage(60, BRUTE, def_zone = limb, blocked = armor, wound_bonus = rand(-20,10), sharpness = SHARP_NONE)
+				C.Knockdown(3 SECONDS)
+				var/atom/throw_target = get_edge_target_turf(C, get_dir(S, get_step_away(C, S)))
+				C.throw_at(throw_target, 3, 6)
+				playsound(get_turf(M), 'sound/effects/hit_kick.ogg', 50, TRUE, -1)
+				M.visible_message("<span class='danger'>[M] is smashed by a large vine!</span>", \
+				"<span class='userdanger'>You are smashed by a large vine!</span>")
+				log_combat(S, M, "aggressively smashed")
+	else //Living but not a carbon? Maybe a silicon? Can't be wounded so have a big chunk of simple bruteloss with no special effects. They can be entangled.
+		M.adjustBruteLoss(75)
+		playsound(get_turf(M), 'sound/weapons/whip.ogg', 50, TRUE, -1)
```

Comment:
`playsound(M` instead of `playsound(get_turf(M)` here and all other uses.

Reply:
Done!